sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/implementation/models/Feature.java

```diff
@@ -24,10 +24,16 @@ public class Feature {
     private Boolean evaluate = true;
```

Comment:
Will we always have this field, if so, we can change the type to `boolean`.

Reply:
If possible, the getter should be `public boolean isEvaluate()`

Reply:
Sorry, I meant if this `evaluate` field will always be present. If so, the `evaluate` could be a primitive type, having two values, true and false. If not, it can be a `Boolean`, having three values, including null.

Reply:
I think if a customer modified the config to be empty it would result in null. That should always be possible.